assets/src/components/AssignmentTable.js

```diff
@@ -368,6 +370,7 @@ function AssignmentTable (props) {
         </Grid>
       </Grid>
       <TableContainer className={classes.container} ref={tableRef}>
+        <Box sx={visuallyHidden} aria-live='polite'>{filteredAssignments.length} assignments are listed</Box>
```

Comment:
I have minor suggestion here for the singular vs plural statement, like:

0 assignment is listed
1 assignment is listed
2 assignments are listed
...

Reply:
I have implemented the change here